code/modules/research/designs/boards/machine_engie.dm

```diff
@@ -14,6 +14,16 @@
 	category = "Machine Boards"
 	build_path = /obj/item/weapon/circuitboard/smes
 
+/datum/design/treadmill
```

Comment:
It is a power generator, it belongs with the engineering designs. This is a petty conversation anyway because there is no designs/board/machine_gym.dm